pkg/network/server.go

```diff
@@ -51,6 +51,9 @@ var (
 	errMaxPeers         = errors.New("max peers reached")
 	errServerShutdown   = errors.New("server shutdown")
 	errInvalidInvType   = errors.New("invalid inventory type")
+
+	ErrSender              = errors.New("P2PNotary contract should be a sender of the fallback transaction")
```

Comment:
Please, add some meaningfull comment to these exported errors, they will be used from other packages. Also, we may want to place them under a separate `var` declaration as a set of exported errors.

Reply:
Fixed